src/program/lwaftr/csv_stats.lua

```diff
@@ -65,17 +88,30 @@ function CSVStatsTimer:tick()
    local elapsed = engine.now() - self.start
    local dt = elapsed - self.prev_elapsed
    self.prev_elapsed = elapsed
-   self.file:write(('%f'):format(elapsed))
+   if not self.hydra_mode then
+      self.file:write(('%f'):format(elapsed))
+   end
```

Comment:
Admittedly the overall approach of conditionally changing the output depending on a flag, instead of doing it via object orientation, is not great: however it would have taken quite a bit of abstraction and decoupling for just a few small changes to the code.